src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -134,6 +134,9 @@ void GcsActorScheduler::CancelOnLeasing(const NodeID &node_id, const ActorID &ac
   auto node_it = node_to_actors_when_leasing_.find(node_id);
   RAY_CHECK(node_it != node_to_actors_when_leasing_.end());
   node_it->second.erase(actor_id);
+  if (node_it->second.empty()) {
+    node_to_actors_when_leasing_.erase(node_it);
```

Comment:
@wumuzi520 If line-138 erase node_id, can the change of line-277 be removed?

Reply:
I am afraid not, node_it->second may be contains more than one actors.

The change of line-138 is just to avoid garbage.